content/en/docs/languages/js/instrumentation.md

```diff
@@ -1192,9 +1192,19 @@ Node.js or Web SDKs.
 
 ## Metrics
 
-To start producing [metrics](/docs/concepts/signals/metrics), you'll need to
-have an initialized `MeterProvider` that lets you create a `Meter`. `Meter`s let
-you create `Instrument`s that you can use to create different kinds of metrics.
+[Metrics](/docs/concepts/signals/metrics) combine individual measurements into
+aggregations, and produce data which is constant as a function of system load.
+The aggregations lack details required to diagnose low level issues, but
+complement spans by helping to identify trends and providing application runtime
+telemetry.
+
+The metrics API defines a variety of instruments. Instruments record
+measurements, which are aggregated by the metrics SDK and eventually exported
+out of process. Instruments come in synchronous and asynchronous varieties.
+Synchronous instruments record measurements as they happen. Asynchronous
+instrument register a callback, which is invoked once per collection, and which
+records measurements at that point in time.
```

Comment:
```suggestion
Synchronous instruments record measurements as they happen. Asynchronous
instruments register a callback that is invoked once per collection, and which
record measurements at that point in time.
```

"at that point in time" isn't as clear as it could be, but it's good enough for now.